src/components/GroceryItem/GroceryItem.js

```diff
@@ -1,12 +1,25 @@
 import React, { useState, useEffect } from 'react';
 import { firestore } from '../../lib/firebase.js';
+import calculateEstimate from '../../lib/estimates';
 
 const GroceryItem = ({ item }) => {
   const groceryItem = firestore.collection('groceryItems').doc(item.id);
   const [isChecked, setChecked] = useState(false);
 
   const oneday = 60 * 60 * 24 * 1000;
 
+  const fromMilliSecToDays = (time) => Math.ceil(time / 86400000);
+
+  const workOutNextPurchDate = (howSoon) => {
```

Comment:
Don't we want to use the calculateEstimate method that you are bringing in from our lib directory here?